cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -1604,8 +1618,6 @@ SwiftGenerator::writeDispatchOperation(::IceInternal::Output& out, const Operati
     const ParameterList inParams = op->inParameters();
     const bool returnsAnyValues = op->returnsAnyValues();
 
-    const string swiftModule = getSwiftModule(getTopLevelModule(op));
```

Comment:
Unused I assume

Reply:
Indeed, a totally dead local variable.
Stumbled across it while reviewing all the calls to `getSwiftModule` (the function that checked for `swift:module`)